FYDP_ModV1/src/version1.java

```diff
@@ -1611,6 +1623,142 @@ else if(t>=24 && t<30) {
 					}
 				}
 
+				//output sheet for each cohort
+				String sheetNames [] = new String[n3-2];
```

Comment:
instead of n3-2 you can use the variable teachingCohort

Reply:
done and done!